metricflow/sql/render/redshift.py

```diff
@@ -21,11 +21,15 @@ def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionR
         arg_rendered = self.render_sql_expr(node.order_by_arg)
         params = arg_rendered.execution_parameters
 
+        approximate_str = (
+            "APPROXIMATE " if node.percentile_args.function_type is SqlPercentileFunctionType.DISCRETE else ""
```

Comment:
Ooof, this doesn't work? https://docs.aws.amazon.com/redshift/latest/dg/r_WF_PERCENTILE_DISC.html

If that's the case, I'd rather validate and throw an error if we're requesting the approx for discrete and the true value otherwise. approx_percentile can be divergent from percentile by a fair bit, depending on the algorithm and underlying distribution of the data, so giving users approx in one case and "true" in the other without making it clear that this is what's happening is best avoided.

I don't think we can do this as a semantic validation on the model, since it's an engine capability thing, but we could throw an informative error here instructing the user to disable the discrete percentile flag in the model. If that proves overly burdensome we could downgrade it to a warning.